photos/db.go

```diff
@@ -148,26 +149,15 @@ SELECT id FROM photos.users WHERE id = $1`
 // findClosestUserByID selects the first user which exists with
 // id >= userID. Returns the found user ID or an error.
 func findClosestUserByID(tx *sql.Tx, userID int) (int, error) {
-	// TODO(spencer): enable finding closest user when DB becomes more performant.
-	/*
-			var id int
-			const selectSQL = `
-		SELECT id FROM photos.users WHERE id >= $1 LIMIT 1;`
-			err := tx.QueryRow(selectSQL, userID).Scan(&id)
-			switch err {
-			case nil:
-				return id, nil
-			default:
-				return 0, err
-			}
-	*/
-	exists, err := userExists(tx, userID)
-	if err != nil {
-		return 0, err
-	} else if !exists {
+	var id int
+	const selectSQL = `
+SELECT id FROM users WHERE id >= $1 ORDER BY id LIMIT 1;
```

Comment:
Interestingly, this sort of query can confuse MySQL https://news.ycombinator.com/item?id=9983374